2020-InfiniteRecharge/src/main/java/frc/robot/Robot.java

```diff
@@ -225,38 +228,38 @@ public void disabledPeriodic()
   public void initialize()
   {
     robotInitialized = true;
-    if(ImuEnable)
+    if(ImuEnable && robotInitialized != false)
     {
       imu.init();
     }
   
-  if(powerCellEnable){
+  if(powerCellEnable && robotInitialized != false){
       powerCell.init(oi);
     }
 
     System.out.println("Initilied");
-    if(chassisEnable)
+    if(chassisEnable && robotInitialized != false)
     {
       driveTrain.Init(oi);
     }
     System.out.println("ChassisEnable: " + chassisEnable);
 
-    if(climberEnable){
+    if(climberEnable && robotInitialized == false){
```

Comment:
Doesn't this check make it so the climber never gets initialized?

Reply:
That has been changed as well